zallet/tests/cmd/migrate_zcash_conf_mainnet.toml

```diff
@@ -16,11 +24,17 @@ identity = ""
 
 [limits]
 
+[note_management]
+
+[outputs]
```

Comment:
Currently every section header is required to be present in the config file (vs individual fields which can almost entirely be omitted if the defaults are fine). Do we want that to remain the case, or should I experiment with making entire sections optional?

I think I'd still keep the section headers uncommented in the example config (as otherwise uncommenting a commented-out field would cause it to apply to a different section, leading to confusing error messages).